2021/08/04 12:34:12 [error] 32#32: *1 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.21.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.21.0.2:9000", host: "127.0.0.1"
2021/08/04 18:37:42 [error] 31#31: *6 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.25.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.3:9000", host: "127.0.0.1"
2021/08/04 19:16:38 [error] 31#31: *14 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.25.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.3:9000", host: "127.0.0.1"
2021/08/04 19:16:38 [error] 31#31: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.25.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.3:9000", host: "127.0.0.1"
2021/08/04 19:16:39 [error] 31#31: *14 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.25.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.3:9000", host: "127.0.0.1"
2021/08/04 19:16:39 [error] 31#31: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.25.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.3:9000", host: "127.0.0.1"
2021/08/04 19:16:39 [error] 31#31: *14 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.25.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.3:9000", host: "127.0.0.1"
2021/08/04 19:16:39 [error] 31#31: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.25.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.3:9000", host: "127.0.0.1"
2021/08/04 19:16:39 [error] 31#31: *14 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.25.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.3:9000", host: "127.0.0.1"
2021/08/05 14:03:34 [error] 31#31: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.25.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.3:9000", host: "127.0.0.1"
2021/08/05 14:03:36 [error] 31#31: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.25.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.25.0.3:9000", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2021/08/05 14:04:48 [error] 31#31: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.25.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.3:9000", host: "127.0.0.1"
2021/08/05 14:46:09 [error] 31#31: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.25.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.3:9000", host: "127.0.0.1"
2021/08/05 14:48:17 [error] 31#31: *11 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.25.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.3:9000", host: "127.0.0.1"
2021/08/06 07:16:16 [error] 32#32: *6 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:16:16 [error] 32#32: *6 FastCGI sent in stderr: "PHP message: [critical] Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\FatalError: "Error: Cannot declare class App\Contriller\TestController, because the name is already in use" at /var/www/src/Controller/TestController.php line 4" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:16:17 [error] 32#32: *6 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:16:17 [error] 32#32: *6 FastCGI sent in stderr: "PHP message: [critical] Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\FatalError: "Error: Cannot declare class App\Contriller\TestController, because the name is already in use" at /var/www/src/Controller/TestController.php line 4" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:16:20 [error] 32#32: *6 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/favicon.ico" (from "http://127.0.0.1/")" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2021/08/06 07:18:32 [error] 32#32: *10 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:18:32 [error] 32#32: *10 FastCGI sent in stderr: "PHP message: [critical] Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\FatalError: "Error: Cannot declare class App\Contriller\DevController, because the name is already in use" at /var/www/src/Controller/DevController.php line 4" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:20:30 [error] 32#32: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:20:30 [error] 32#32: *13 FastCGI sent in stderr: "PHP message: [critical] Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\FatalError: "Error: Cannot declare class App\Contriller\DevController, because the name is already in use" at /var/www/src/Controller/DevController.php line 4" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:20:54 [error] 32#32: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:20:54 [error] 32#32: *13 FastCGI sent in stderr: "PHP message: [critical] Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\FatalError: "Error: Cannot declare class App\Contriller\DevController, because the name is already in use" at /var/www/src/Controller/DevController.php line 4" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:22:22 [error] 32#32: *18 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:22:22 [error] 32#32: *18 FastCGI sent in stderr: "PHP message: [critical] Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\FatalError: "Error: Cannot declare class App\Contriller\DevController, because the name is already in use" at /var/www/src/Controller/DevController.php line 4" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:26:17 [error] 32#32: *20 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:26:17 [error] 32#32: *20 FastCGI sent in stderr: "PHP message: [critical] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\ControllerDoesNotReturnResponseException: "The controller must return a "Symfony\Component\HttpFoundation\Response" object but it returned null. Did you forget to add a return statement somewhere in your controller?" at /var/www/src/Controller/DevController.php line 13" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:26:45 [error] 32#32: *20 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index".
PHP message: [critical] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\ControllerDoesNotReturnResponseException: "The controller must return a "Symfony\Component\HttpFoundation\Response" object but it returned a string ("It's the Constructor")." at /var/www/src/Controller/DevController.php line 13" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:28:54 [error] 32#32: *24 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index".
PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [critical] Uncaught PHP Exception TypeError: "App\Controller\DevController::index(): Return value must be of type Symfony\Component\HttpFoundation\Response, string returned" at /var/www/src/Controller/DevController.php line 15" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:28:54 [error] 32#32: *24 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: 2021-08-06T07:28:54+00:00 [critical] Uncaught Error: App\Controller\DevController::index(): Return value must be of type Symfony\Component\HttpFoundation\Response, string returned" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:29:50 [error] 32#32: *24 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index".
PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [critical] Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\UndefinedMethodError: "Attempted to call an undefined method named "json" of class "App\Controller\DevController"." at /var/www/src/Controller/DevController.php line 15" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:29:50 [error] 32#32: *24 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: 2021-08-06T07:29:50+00:00 [critical] Uncaught Error: Call to undefined method App\Controller\DevController::json()" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:33:11 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:33:31 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:34:29 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:34:29 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [critical] Uncaught PHP Exception TypeError: "App\Controller\DevController::index(): Return value must be of type Symfony\Component\HttpFoundation\Response, none returned" at /var/www/src/Controller/DevController.php line 17" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:34:29 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: 2021-08-06T07:34:29+00:00 [critical] Uncaught Error: App\Controller\DevController::index(): Return value must be of type Symfony\Component\HttpFoundation\Response, none returned" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:34:44 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:34:44 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [critical] Uncaught PHP Exception TypeError: "App\Controller\DevController::index(): Return value must be of type Symfony\Component\HttpFoundation\Response, none returned" at /var/www/src/Controller/DevController.php line 17" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:34:44 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: 2021-08-06T07:34:44+00:00 [critical] Uncaught Error: App\Controller\DevController::index(): Return value must be of type Symfony\Component\HttpFoundation\Response, none returned" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:34:54 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:35:26 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:35:26 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: [critical] Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Controller class "App\Controller\DevController" cannot be called without a method name. You need to implement "__invoke" or use one of the available methods: "index"." at /var/www/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:35:43 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [critical] Uncaught PHP Exception Symfony\Component\Config\Exception\LoaderLoadException: "The file "/var/www/src/../config/routes.yaml" does not contain valid YAML: Unable to parse in "/var/www/src/../config/routes.yaml" at line 6 (near ":") in /var/www/src/../config/routes.yaml (which is being imported from "/var/www/src/Kernel.php")." at /var/www/vendor/symfony/config/Loader/FileLoader.php line 174" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:36:08 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:36:08 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: [critical] Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Controller class "App\Controller\DevController" cannot be called without a method name. You need to implement "__invoke" or use one of the available methods: "index"." at /var/www/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:36:42 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:36:42 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: [critical] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\ControllerDoesNotReturnResponseException: "The controller must return a "Symfony\Component\HttpFoundation\Response" object but it returned null. Did you forget to add a return statement somewhere in your controller?" at /var/www/src/Controller/DevController.php line 11" while reading upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:37:29 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:38:13 [error] 32#32: *29 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/dev"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET /dev HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:39:48 [error] 32#32: *44 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/dev"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET /dev HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
2021/08/06 07:40:13 [error] 32#32: *44 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "dev"" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET /dev HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1"
